Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -225,6 +225,7 @@ script:
     arguments:
     - name: ids
       description: A comma separated list of one or more IDs to resolve.
+      required: true
```

Comment:
this is breaking BC, are you sure this is required?

Reply:
the command will not work without it. So I can set it as a non required param but the API requires at least one detection id to update.
Should I keep it as it was?

Reply:
yes, keep the required